molecule/default/tests/test_verify_mbu_installed.py

```diff
@@ -0,0 +1,53 @@
+import os
+import pytest
+import testinfra.utils.ansible_runner
+
+testinfra_hosts = testinfra.utils.ansible_runner.AnsibleRunner(
+    os.environ['MOLECULE_INVENTORY_FILE']).get_hosts('os-infra_hosts')[:1]
+
+
+backup_folder_on_host = '/openstack/backup'
+backup_folder_on_container = '/var/backup'
+
+
+@pytest.mark.incremental
+class TestVerifyMBUInstall(object):
+    @pytest.mark.test_id('f0ef4020-a16f-11e8-97f3-fe0851898bb5')
+    @pytest.mark.jira('asc-262')
+    def test_verify_backup_folder_on_host(self, host):
+        assert host.file(backup_folder_on_host).exists
+
+    @pytest.mark.test_id('4a7ed621-a1a3-11e8-8948-6a0003552100')
+    @pytest.mark.jira('asc-262')
+    def test_veiry_backup_folder_on_container(self, host):
+        cmd1 = "ls {}".format(backup_folder_on_host)
+        output = host.run(cmd1).stdout
+        containers = output.split('\n')
+
+        # Fail the test if there is ::no container in the backup folder
+        print containers
+        assert (len(containers) > 1)
```

Comment:
Should this assertion be `> 0` instead? Are 2 or more containers expected?

Reply:
The command `ls` will always result one `\n`, so the `split('\n')` creates a list with len(containers) == 1 if the backup directory is empty